search.php

```diff
@@ -6,49 +6,110 @@
  * @package shiro
  */
 
-get_header(); ?>
-
-<?php
+get_header();
 
 $wmf_results_copy = get_theme_mod( 'wmf_search_results_copy', /* translators: 1. search query */ __( 'Search results for %s', 'shiro-admin' ) );
 
 $template_args = array(
 	/* translators: Query that is currently being searched */
-	'h1_title' => sprintf( __( $wmf_results_copy, 'shiro' ), get_search_query() ),
+	'h1_title' => sprintf( __( $wmf_results_copy, 'shiro' ), get_search_query() ), // phpcs:ignore WordPress.WP.I18n.NonSingularStringLiteralText
```

Comment:
Yeah, I guess this string shouldn't be getting re-translated, since the default string is already translated above.